README.md

```diff
@@ -148,9 +153,7 @@ shell's `$PATH`.
 [Postgraphile](https://www.graphile.org/postgraphile/) is used to expose GraphQL endpoints for our database schemas, this is described in detail [here](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/postgraphile.md).
 
 ### Tests
-- Set the ipc path to a Kovan node either by:
-    - replacing the empty `ipcPath` in the `environments/testing.toml` with a path to a full node's eth_jsonrpc endpoint (e.g. local geth node ipc path or infura url)
-    - Or, setting the CLIENT_IPCPATH environment variable
+- Set the ipc path to a Kovan node either by setting the CLIENT_IPCPATH environment variable.
```

Comment:
Not introduced by this PR but maybe we could remove the reference to Kovan since all integration tests point at mainnet now? Also seems like we can delete the word "either"